src/convertToObject.js

```diff
@@ -6,7 +7,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((prev, el) => {
+      const [property, value] = el
+        .split(':')
+        .map((elem) => elem.trim());
+
+      prev[property] = value;
+
+      return prev;
```

Comment:
The function `convertToObject` correctly splits the string by semicolons and uses `reduce` to build the object. However, it does not handle empty or invalid declarations, which may cause the function to add undefined keys or values to the result object. Consider filtering out any elements that don't contain a colon before the `reduce` operation to ensure only valid CSS declarations are processed.